pkg/registry/batch/job/strategy_test.go

```diff
@@ -44,6 +44,7 @@ func TestJobStrategy_PrepareForUpdate(t *testing.T) {
 	validSelector := getValidLabelSelector()
 	validPodTemplateSpec := getValidPodTemplateSpecForSelector(validSelector)
 
+	failedReplacementPolicy := batch.Failed
```

Comment:
non-blocking nit: Can we also here use `podReplacementPolicyPtr` function?

Reply:
added.